docs/modules/ROOT/pages/property-source-config.adoc

```diff
@@ -7,12 +7,11 @@ an `application-profile.properties` or `application-profile.yaml` file that cont
 application or Spring Boot starters. You can override these properties by specifying system properties or environment
 variables.
 
-To enable this functionality you need to set `spring.config.import=kubernetes:` in your application's configuration properties.
+To enable this functionality you need to set the `spring.config.import` application configuration property to `kubernetes:` (escape with quotes when using yaml eg. `"kubernetes:"`).
```

Comment:
I went back/forth on the verbiage and am not sold on the current incantation in this proposal (ie open to suggestions).